test/rules/accessibility/required-wrapper-accessibility.spec.js

```diff
@@ -0,0 +1,110 @@
+                       </div>
+                    </div>`;
+        var messages = HTMLHint.verify(code, ruleOptions);
+        expect(messages.length).to.be(0);
+    });
+
+    it('div with "requiredWrapper" binding with validationMessage and element that points to it inside an inner div should not result in an error', function () {
+        var code = `<div data-bind="requiredWrapper:firstName">
+                       <span class="validationMessage" id="${messagePrefix}input1" aria-live="assertive"></span>
+                       <div>
+                       <input id='firstName' data-bind='addDescription:"${messagePrefix}input1"'/>
+                       </div>
+                    </div>`;
+        var messages = HTMLHint.verify(code, ruleOptions);
+        expect(messages.length).to.be(0);
+    });
+
+    it('div with "requiredWrapper" binding with validationMessage and element with multiple description should not result in an error', function () {
+        var code = `<div data-bind="requiredWrapper:firstName">
+                       <span class="validationMessage" id="${messagePrefix}input1" aria-live="assertive"></span>
+                       <input id='firstName' data-bind='addDescription:"${messagePrefix}input1  desc"'/>
+                    </div>`;
+        var messages = HTMLHint.verify(code, ruleOptions);
+        expect(messages.length).to.be(0);
+    });
+
+    it('div with "requiredWrapper" binding with validationMessage and element that points to it in other div should result in an error', function () {
+        var code = `<div data-bind="requiredWrapper:firstName">
+                       <span class="validationMessage" id="${messagePrefix}input1" aria-live="assertive"></span>
+                    </div>
+                    <div data-bind="requiredWrapper:firstName">
+                       <input id='firstName' data-bind='addDescription:"${messagePrefix}input1"'/>
+                    </div>`;
+        var messages = HTMLHint.verify(code, ruleOptions);
+        expect(messages.length).to.be(2);
+        expect(messages[0].rule.id).to.be(ruldId);
+        expect(messages[0].line).to.be(3);
+        expect(messages[0].col).to.be(21);
+    });
+    it('div with "requiredWrapper" binding with validationMessage and element that points to it in nested "requiredWrapper" div should result in an error', function () {
+        var code = `<div data-bind="requiredWrapper:firstName">
+                       <span class="validationMessage" id="${messagePrefix}input1" aria-live="assertive"></span>
+                       <div data-bind="requiredWrapper:firstName">
+                       <input id='firstName' data-bind='addDescription:"${messagePrefix}input1"'/>
+                    </div>
+                    </div>`;
+        var messages = HTMLHint.verify(code, ruleOptions);
+        expect(messages.length).to.be(0);
+    });
+
```

Comment:
Add a regression test for div without requiredWrapper.
expected result: no errors